src/zzz_od/application/world_patrol/operation/world_patrol_run_route.py

```
@@ -117,112 +134,171 @@ def run_op(self) -> OperationRoundResult:
+                if time_since_last_pos > self.TIME_THRESHOLD_FOR_FAIL_NO_POS:
+                    log.error('长时间无法计算坐标，任务失败')
+                    return None  # 触发上层逻辑失败
+                if time_since_last_pos > self.TIME_THRESHOLD_FOR_STUCK_NO_POS and self._get_rid_of_stuck():
+                    log.error('脱困次数超限，重启路线')
+                    return None  # 触发上层逻辑失败
+                if time_since_last_pos > self.TIME_THRESHOLD_FOR_STOP_NO_POS:
+                    self.ctx.controller.stop_moving_forward()
+        if self.no_pos_start_time == self.last_screenshot_time:  # 刚开始无法获取坐标
+            self.ctx.controller.turn_vertical_by_distance(300)  # 轻微上抬视角
+        else:
+            self.no_pos_start_time = 0  # 成功获取坐标，重置计时器
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)